sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -0,0 +1,57 @@
+/*
+
+This query simply copies data from a table in the restricted suggest-searches-prod
+project to shared-prod, but throws an error on empty input in order to signal an
+upstream delay or error.
+
+In order to produce the error, it includes aggregations and join steps.
+
+*/
+WITH sanitized_impressions AS (
+  SELECT
+    *
+  FROM
+    `suggest-searches-prod-a30f.sanitized.suggest_impression_sanitized_v3`
```

Comment:
This and similar references should be changed to `moz-fx-data-shared-prod.search_terms_derived.merino_log_sanitized_v3`.

In addition, since the sanitizer service won't be writing to this table anymore (airflow will) the sanitizer should have its write access removed via `metadata.yaml`.

Reply:
Done and removed!